src/WebJobs.Script.WebHost/Diagnostics/FastLogger.cs

```diff
@@ -7,16 +7,29 @@
 using Microsoft.Azure.WebJobs.Host;
 using Microsoft.Azure.WebJobs.Host.Loggers;
 using Microsoft.Azure.WebJobs.Logging;
+using Microsoft.Azure.WebJobs.Script.Description;
+using Microsoft.Azure.WebJobs.Script.Diagnostics;
 using Microsoft.WindowsAzure.Storage;
 
 namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
 {
     // Adapter for capturing SDK events and logging them to tables.
     internal class FastLogger : IAsyncCollector<FunctionInstanceLogEntry>
```

Comment:
Recommend we rename this class to FunctionInstanceLogger which better describes its expanded role